### DISCUSSION TITLE
How matching 2 test cases in 1 autotest in launches

### DISCUSSION THREAD

**Discussion by mrrinatino** [allure-framework/allure2#2494] [TestOps]
Hello! I would like to ask. This approach is quite often used in my project for automation. There are several manual test cases, the checks for these cases are combined into one automated test. Currently, we are exploring the allure interface for migration and looking for a solution on how to match such automated tests with manual test cases after running.

Even in the code provided below, after running the automated tests, only one of the specified allure.id changes its status to automated, the second one is simply ignored and remains manual. Is there anything that can be done here, or does Allure rigidly bind to one test case?

    test(`Test with 2 allure.id`, async ({
      page,
    }) => {
      allure.tag(TESTIT_LABELS.PRIORITY + '_' + TESTIT_PRIORITY.HIGHEST);
      allure.feature(FEATURE_LABELS.ADMIN_ROLES);
      allure.id('186469');
      allure.id('186553');
    });

playwright/typeScript

**Comment by baev** [comment]
If you are willing to create a single autotest for multiple manual test cases, the approach would be to create a new automated test case and link it to manual test cases using relations.

**Comment by mrrinatino** [comment]
Are there any instructions on what needs to be done? In that case, after running the automated test, will all associated manual test cases be marked as automated?
